example/main/demo_esp_littlefs.c

```diff
@@ -37,7 +48,14 @@ void app_main(void)
 
         printf("silicon revision %d, ", chip_info.revision);
 
-        printf("%dMB %s flash\n", spi_flash_get_chip_size() / (1024 * 1024),
+        uint32_t size_flash_chip = 0;
+#if ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 0, 0)
+        esp_flash_get_size(NULL, &size_flash_chip);
```

Comment:
This function is available since IDF v4.0. Since the oldest non-EOL version of IDF is currently v4.1, I think you could leave this as the only code path.

Reply:
addressed in f99e2b2